src/huggingface/forte/huggingface/token_classification.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper of the Token Classification models on HuggingFace platform (context
+understanding)
+"""
+import importlib
+import re
+from typing import Dict, Set
+from transformers import pipeline
+
+from forte.common import Resources
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.processors.base import PackProcessor
+from forte.common import ProcessorConfigError
+from forte.utils.tagging_scheme import bio_merge
+
+__all__ = [
+    "TokenClassification",
+]
+
+
+class TokenClassification(PackProcessor):
+    r"""Wrapper of the models on HuggingFace platform with pipeline tag of
+    `ner`.
+    https://huggingface.co/models?pipeline_tag=token-classification
+    This wrapper could take any model name on HuggingFace platform with pipeline
+    tag of `ner` in configs to make prediction on the user
+    specified entry type in the input pack and the prediction result goes to the
+    user specified attribute name of that entry type in the output pack. User
```

Comment:
This docstring explanation is hard to follow. Can you explain what it does in simpler language? You can postpone the explanation of how to use attribute names and entry types, and focus on a high-level explanation first.

You can provide examples instead of using plain text to explain the configs (use a code block here). And then you can refer readers to the `default_config` function for more information. 

Put yourself into the user's hat, see if you can understand the usage of this by reading the docstring?